src/worker.js

```diff
@@ -43,16 +51,19 @@ module.exports = async function () {
 
     const relativeFilePath = Helpers.getRelativePath(fileDir, filePath, config, projectPath)
 
-    const cliEngineOptions = Helpers.getCLIEngineOptions(
-      type, config, rules, relativeFilePath, fileDir, configPath
-    )
+    const cliEngineOptions = Helpers
+      .getCLIEngineOptions(type, config, rules, relativeFilePath, fileDir, configPath)
 
     let response
     if (type === 'lint') {
-      const report = lintJob({ cliEngineOptions, contents, eslint, filePath })
+      const report = lintJob({
+        cliEngineOptions, contents, eslint, filePath
+      })
       response = report.results.length ? report.results[0].messages : []
     } else if (type === 'fix') {
-      response = fixJob({ cliEngineOptions, contents, eslint, filePath })
+      response = fixJob({
+        cliEngineOptions, contents, eslint, filePath
+      })
```

Comment:
This (and others like it) is a fairly annoying change, since it breaks a single, understandable line into three lines. What linting rule is requiring it?  Do you like having object literals always broken apart like this?

Reply:
It's being controlled by [object-curly-newline](https://eslint.org/docs/rules/object-curly-newline), and I'm not sure I like it.

The current configuration of this rule can be found [here](https://github.com/airbnb/javascript/blob/9dee84796bfcf3ab88cd4603869c7c7c6ada13b1/packages/eslint-config-airbnb-base/rules/style.js#L344-L349). The critical part looking to be the `minProperties: 4` bit, which we are _just_ hitting on almost every case.

Should we override that rule here?

Reply:
Yeah, I think I'd be tempted to set it to `5`, or just turn `minProperties` off and only set `"consistent": true`.  I'm not really sure I see the benefit of setting a minProperties at all.

To be clear, I'm proposing:

```js
    'object-curly-newline': ['error', {
      ObjectExpression: { consistent: true },
      ObjectPattern: { consistent: true }
    }],
```

What do you think?

Reply:
Tried it out, I think I like that, our `4` level uses aren't that bad, and just setting that to 5 allows it to still catch bigger ones.